src/main/java/com/example/java/maven/game2048/MessagePrinter.java

```diff
@@ -1,4 +1,18 @@
 package com.example.java.maven.game2048;
 
 public class MessagePrinter {
+
+    private MessagePrinter() {
+    }
+
+    public static void printInstructionsBeforeGame(){
+        String instructions = ("INSTRUCTIONS:" + '\n' +
```

Comment:
Move this instruction to [constant](https://www.thoughtco.com/constant-2034049) in game.
For example:
```java
private static final String INSTRUCTIONS = "YOUR_STRING_SHOULD_BE_HERE"
```

Reply:
All command can be moved to constants

Reply:
Done